coursier.bzl

```diff
@@ -798,6 +798,8 @@ def make_coursier_dep_tree(
         cmd.extend(["--repository", repository["repo_url"]])
         if "credentials" in repository:
             cmd.extend(["--credentials", utils.repo_credentials(repository)])
+    for credential in utils.netrc_credentials(get_home_netrc_contents(repository_ctx)):
```

Comment:
We probably only want to set this if we haven't already set the credentials (see the line immediately above this one!), and if users ask for us to add the credentials. There's an open issue that `coursier` leaves credentials in URLs, so I'm loathe to pull things from `.netrc` only to expose them accidentally in a PR. Could you please make this behaviour conditional and opt-in?

Reply:
Yes, we can add a flag for it.